media/CircleGraph/view.js

```diff
@@ -542,13 +542,26 @@ view.View = class {
     }
   }
 
+  /**
+   * @brief called when it's needed to change scroll behaviour
+   */
+  setScrollToSelected(value) {
+    this._scrollToSelected = value;
+  }
+
   /**
    * @brief toggleSelect will select or toggle select with CtrlKey down
    * @param viewNode view.Node instance
    * @note  works on host mode is viewMode.selector
    */
   toggleSelect(viewNode) {
-    if (viewNode && this._host._mode === viewMode.selector) {
+    if (!viewNode) {
+      return;
+    }
```

Comment:
Please do not change like this. I don't like return in the middle of a method.

Reply:
ok.

Reply:
I don't mind. Let me know when it's ready :-D

Reply:
@dayo09 @seanshpark 
fixed